storm-core/src/jvm/org/apache/storm/utils/Utils.java

```diff
@@ -1050,6 +1083,12 @@ public String getBackupConnectionString() throws Exception {
         }
     }
 
+    public static void testSetupBuilder(CuratorFrameworkFactory.Builder
```

Comment:
This was originally in a separate class to separate it out only for testing purposes.  I am OK with having this here, but lest add some javadocs to say this is only for testing.  The name should give a lot of a warning, but a full explanation would be good.